src/native/corehost/deps_format.cpp

```diff
@@ -96,20 +96,21 @@ void deps_json_t::reconcile_libraries_with_targets(
                 entry.asset = asset;
                 entry.asset.name = asset_name;
 
-                m_deps_entries[i].push_back(entry);
+                m_deps_entries[i].push_back(std::move(entry));
 
                 if (trace::is_enabled())
                 {
+                    const deps_entry_t& parsed = m_deps_entries[i].back();
```

Comment:
Maybe if push_back/move happens after `if (trace::is_enabled())` block, we would not need `parsed` ?